src/inference/support/hmc_core.jl

```diff
@@ -251,12 +253,13 @@ function find_good_eps(model, spl::Sampler{T}, vi::VarInfo) where T
 
     # Heuristically find optimal ϵ
     while (iter_num <= 12)
```

Comment:
Maybe we could make the 12 to be a keyword argument of find_good_eps?

Reply:
Sure.